app/controllers/application_controller.rb

```diff
@@ -1,7 +1,15 @@
 class ApplicationController < ActionController::Base
-  before_action :set_current_user
+  before_action :configure_permitted_parameters, if: :devise_controller?
 
-  def set_current_user
-    Current.user = User.first
+  # private
+
+  # def after_sign_out_path_for(resource_or_scope)
+  #     root_path
+  # end
```

Comment:
* Kindly remove all commented code as it makes your code base bulky and unprofessional 🙏